wandb/sdk/internal/internal_api.py

```diff
@@ -1404,32 +1420,79 @@ def create_run_queue(
             raise UnsupportedError(
                 "default resource configurations are not supported by this version of wandb server."
             )
+        if not supports_prioritization and prioritization_mode is not None:
+            raise UnsupportedError(
+                "launch prioritization is not supported by this version of wandb server."
```

Comment:
could you know which minimal version will support it? just thinking of a more actionable error message. maybe at least let people know how if they can update to latest or what not